registry.k8s.io/images/k8s-staging-provider-aws/images.yaml

```diff
@@ -66,7 +66,11 @@
     "sha256:d9e91f03babf5a89a3bed4b6f4b9ad47658e713784b431f25813b3c1938716c6": ["v1.22.1"]
     "sha256:1669262267c5b89e94c67ab400d5b8fde95faa60066a1c04082e85e3e76710a9": ["v1.23.0"]
     "sha256:fe9d99c75151a656abbc1dd44312e774bea7ec3da5084dfc7ab824e9f72291cf": ["v1.23.1"]
+    "sha256:8af537574034e1e82557d408ca0471d4892c7f523f1d2f7255718c632f7c9dd6": ["v1.23.2"]
+    "sha256:fc12045699ac3fd986eac1396978a99a602a8d4e0d421881a5dbd62355231039": ["v1.23.2-linux-arm64-al2"]
```

Comment:
This tag needs to be `v1.23.2-a1compat` for the chart to pick it up

Same thing with the other image